packages/notifi-dapp-example/src/components/DestinationErrorMessage.tsx

```diff
@@ -21,17 +20,20 @@ export const DestinationErrorMessage: React.FC<Props> = ({
     <div>
       {onClick !== undefined ? (
         isButton ? (
```

Comment:
Just wondering if it is possible to avoid this multiple nesting structure. example below:
```
if (!onClick) return <> ...</>
return <>
  {isButton ? XXX : XXX}
</>